src/WormholeJoin.sol

```diff
@@ -185,44 +185,41 @@ contract WormholeJoin {
             vat.slip(ilk, address(this), int256(amtToGenerate));
             vat.frob(ilk, address(this), address(this), address(this), int256(amtToGenerate), int256(amtToGenerate));
         }
-        uint256 postFeeAmount = amtToTake - fee;
-        daiJoin.exit(bytes32ToAddress(wormholeGUID.receiver), postFeeAmount - operatorFee);
+        postFeeAmount = amtToTake - fee;
+        daiJoin.exit(bytes32ToAddress(wormholeGUID.receiver), postFeeAmount);
 
         if (fee > 0) {
             vat.move(address(this), vow, fee * RAY);
         }
-        if (operatorFee > 0) {
-            vat.move(address(this), bytes32ToAddress(wormholeGUID.operator), operatorFee * RAY);
-        }
 
-        emit Withdraw(hashGUID, wormholeGUID, amtToTake, maxFeePercentage, operatorFee);
+        emit Withdraw(hashGUID, wormholeGUID, amtToTake, maxFeePercentage);
     }
 
     /**
     * @dev External authed function that registers the wormwhole and executes the mint after
     * @param wormholeGUID Struct which contains the whole wormhole data
     * @param maxFeePercentage Max percentage of the withdrawn amount (in WAD) to be paid as fee (e.g 1% = 0.01 * WAD)
-    * @param operatorFee The amount of DAI to pay to the operator
+    * @return The amount of DAI sent to the receiver after taking out fees
     **/
-    function requestMint(WormholeGUID calldata wormholeGUID, uint256 maxFeePercentage, uint256 operatorFee) external auth {
+    function requestMint(WormholeGUID calldata wormholeGUID, uint256 maxFeePercentage) external auth returns (uint256) {
```

Comment:
We might want to use the same approach than in the internal function, call the variable in the header and then use the assignment. Apart from keeping it standard it is nicer when you generate the abi that you get the name of the variable.